webpack.config.js

```diff
@@ -65,11 +65,28 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader',
+					"webpack-append?@import 'assets/stylesheets/shared/component';",
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
```

Comment:
Yes, we limited it only to one file to validate it works properly. We plan to provide general solution in the follow up PR.

As mentioned in a different place, we still need to cover the following items:
* We should import style.scss file only when it's present in the folder where React component is located
* RTL support is missing
* SSR support